app/faculty/faculty-design.css

```
@@ -0,0 +1,393 @@
+  --fd-surface:   #FFFFFF;
+  --fd-surface-2: #FBF6F2;
+  --fd-line:      #EFE2DA;
+  --fd-line-2:    #F5EBE4;
+  --fd-text:      #1B1A18;
+  --fd-text-2:    #5C564F;
+  --fd-text-3:    #8C8580;
+
+  --fd-shadow-sm: 0 1px 2px rgba(20, 18, 14, 0.04), 0 1px 1px rgba(20, 18, 14, 0.03);
+  --fd-shadow-lg: 0 18px 40px rgba(20, 18, 14, 0.08), 0 4px 10px rgba(20, 18, 14, 0.04);
+
+  --fd-mono: "JetBrains Mono", ui-monospace, "SF Mono", Menlo, monospace;
```

Stylelint (17.10.0)

[error] Expected "Menlo" to be "menlo" (value-keyword-case)

(value-keyword-case)